app.py

```diff
@@ -250,6 +250,7 @@ def check_bronze_auth(flash_it=False):
 def check_bronze_kyc_level():
     if not hasattr(current_user, 'bronze_data') or not current_user.bronze_data:
         return False;
+    add_update_bronze_data(current_user)
```

Comment:
This would probably be better like:

```
    # if not kyc validated check again with bronze to see if updated
    if not current_user.bronze_data.kyc_validated:
        add_update_bronze_data(current_user)
    return current_user.bronze_data.kyc_validated
```

Reply:
adjusted the code.